react-components/src/components/NodeCacheProvider/FdmNodeCache.ts

```diff
@@ -203,13 +225,18 @@ export class FdmNodeCache {
   }
 
   private async getAndCacheRevisionToEdgesMap(
-    modelRevisionIds: ModelRevisionId[]
+    modelRevisionIds: ModelRevisionId[],
+    fetchViews: boolean = false
```

Comment:
I don't think this should be default `false`, it should be required as argument from the caller

Reply:
Removed default value.